docs/source/manager.md

```diff
@@ -160,7 +160,7 @@ Add following task definition to Cluster spec:
       interval: "1d"
 ```
 
-For full task definition configuration consult [Scylla Cluster CRD](scylla-cluster-crd.md).
+For full task definition configuration consult [Scylla Cluster CRD](api-reference/groups/scylla.scylladb.com/scyllaclusters.rst).
```

Comment:
nit: not an addition but s/Scylla Cluster CRD/ScyllaCluster CRD
Also maybe leaving the link text empty wouldn't be a bad idea?
![image](https://github.com/scylladb/scylla-operator/assets/32443348/cb6f3efc-598c-46e7-915c-6841b7941fb6)

Reply:
> Also maybe leaving the link text empty wouldn't be a bad idea?

possibly, but on the other hand some changes in the other file might make this look worse over time without us noticing

Reply:
space fixed